lib/map.jsx

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+
+var MapComponent = React.createClass({
+
+    propTypes: {
+      accessToken: React.PropTypes.string.isRequired,
+      mapId: React.PropTypes.string.isRequired,
+    },
+
+    componentDidMount: function () {
+      console.log('here1')
+      // Provide your access token
+      L.mapbox.accessToken = this.props.accessToken;
+
+      // Create a map in the div #map
+      var map = L.mapbox.map('map', this.props.mapId);
```

Comment:
We might want to assign the map to `this.map` so the component can use it later?